src/js/control-bar/time-controls/current-time-display.js

```diff
@@ -70,12 +70,12 @@ class CurrentTimeDisplay extends TimeDisplay {
 }
 
 /**
- * The text that should display over the `CurrentTimeDisplay`s controls. Added to for localization.
+ * The text that is added to the `CurrentTimeDisplay` for screen reader users.
  *
  * @type {string}
  * @private
  */
-CurrentTimeDisplay.prototype.controlText_ = 'Current Time';
```

Comment:
I'm thinking we should leave the controlText_ property in case anyone is relying on it but internally ignore it and only use labelText_

Reply:
It's confusing, because these components don't extend `Button` or `ClickableComponent` which does have that concept of `controlText_`, and because these *aren't* controls, they're information displays.

It looks like `controlText_` was added when the common `time-display.js` was created. Which version was that? If it wasn't 6.0, then it doesn't seem like anyone ought to be "relying on it", since it's private.